GatherPress/gatherpress
# Updated "past" to "passed"

### Description of the Change
I have updated the following string: "The event has already past." to "The event has already passed.". Tests + translation files were also updated. This string is rendered in the backend when an event has already occurred.

<img width="615" alt="Screenshot 2024-04-06 at 9 46 56 AM" src="https://github.com/GatherPress/gatherpress/assets/28283226/e9766deb-fa84-4050-ab7e-b5210b707eeb">

### How to test the Change
Create/open past event and confirm the new string displays as a warning message to the user.

### Changelog Entry
> Changed - Warning message nomenclature when a user views a past event in the backend

### Credits
@stephenerdelyi 

### Checklist:
